source/includes/connection-snippets/x509-snippets/java-connection.java

```diff
@@ -0,0 +1,25 @@
+
+// begin x509 connection
+import java.util.Arrays;
+import org.bson.Document;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.MongoCredential;
+import com.mongodb.ServerAddress;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoIterable;
+
+
+System.setProperty("javax.net.ssl.trustStore","/etc/certs/mongodb/client.truststore");
+System.setProperty("javax.net.ssl.trustStorePassword","mypass");
+
+System.setProperty("javax.net.ssl.keyStore", "/etc/certs/mongodb/client.keystore");
+System.setProperty("javax.net.ssl.keyStorePassword", "mypass");
+String subject = "<your_subject>"; // CN=ChrisChoClient,OU=TestClientCertificateOrgUnit,O=TestClientCertificateOrg,L=TestClientCertificateLocality,ST=TestClientCertificateStateC=US
```

Comment:
Definitely need to remove this example subject comment from all of the samples. If you really want to put a comment in there, I'd suggest linking to the manual page I mentioned below where it actually describes how to extract your subject in a way that will allow a connection (there are many alternative ways to extract the subject s.t. mongod will reject it)

Reply:
Fixed - replaced with `// x509 client certificate subject`